sal-service/src/main/java/org/ow2/proactive/sal/service/service/MonitoringService.java

```diff
@@ -53,12 +57,19 @@ public class MonitoringService {
     @Autowired
     private RepositoryService repositoryService;
 
+    private String baguetteIp;
+
+    private static final int BAGUETTE_PORT = 8111;
```

Comment:
baguette port ?

Reply:
lool This is the terminology used by EMS monitoring component (implemented by partners).